src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemDecisionsPageViewModel.kt

```diff
@@ -1,13 +1,24 @@
 package nl.avisi.structurizr.site.generatr.site.model
 
 import com.structurizr.model.SoftwareSystem
+import nl.avisi.structurizr.site.generatr.hasContainerDecisions
 import nl.avisi.structurizr.site.generatr.hasDecisions
+import nl.avisi.structurizr.site.generatr.normalize
 import nl.avisi.structurizr.site.generatr.site.GeneratorContext
 
 class SoftwareSystemDecisionsPageViewModel(generatorContext: GeneratorContext, softwareSystem: SoftwareSystem) :
     SoftwareSystemPageViewModel(generatorContext, softwareSystem, Tab.DECISIONS) {
+
     val decisionsTable = createDecisionsTableViewModel(softwareSystem.documentation.decisions) {
         "$url/${it.id}"
     }
-    val visible = softwareSystem.hasDecisions()
+
+    private val visibleContainerDecisions = softwareSystem.hasContainerDecisions()
+    val visibleSoftwareSystemDecisions = softwareSystem.hasDecisions()
+
+    val visible = visibleSoftwareSystemDecisions or visibleContainerDecisions
+    val visibleOnlyContainersDecisions = !visibleSoftwareSystemDecisions and visibleContainerDecisions
```

Comment:
I prefer a different style of naming here:

```suggestion
    private val containerDecisionsVisible = softwareSystem.hasContainerDecisions()
    val softwareSystemDecisionsVisible = softwareSystem.hasDecisions()

    val visible = softwareSystemDecisionsVisible or containerDecisionsVisible
    val onlyContainersDecisionsVisible = !softwareSystemDecisionsVisible and containerDecisionsVisible
```